client/src/locale/he_IL.js

```diff
@@ -17,9 +17,9 @@ const HEBREW_TRANSLATION = {
 	vocabulary: 'מילון',
 	whoWeAre: 'מי אנחנו',
 	meirimTitle: 'מידע תכנוני ואקטיביזם עירוני',
-	newAlert: 'תוכניות בניה',
-	newAlertTree: 'רשיונות כריתה של עצים', 
-	emailAddress: 'כתובת דוא"ל',
+	newAlert: '🏠 תוכניות בניה',
```

Comment:
Can you please add this to the Arabic translation?

Reply:
👍